proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
```

Comment:
I do agree that the MPWG proposal does continue the model used by MP.  But, this statement implies that the multiple independent implementations would not be possible if we reported through the CN4J WG...  I don't think there's anything in the CN4J WG proposal that would prevent MP from continuing their multiple independent impls in a timely manner...

Reply:
This is just waffle it does not continue the model as it brings in a working group with governance structures.

Reply:
Not waffle. Flattens spec committee and steering committee.

Reply:
It was the final bullet point I was saying is waffle as it is not addressing the proposals under review.

Reply:
Fair point, @smillidge.  The pro/con article needs to stick the facts as presented in the two proposals.  If more is being read into the proposals then this doc is either waffling or the proposals are missing some data.